src/Result.php

```diff
@@ -45,29 +45,29 @@ public function getErrors(): array
     }
 
     /**
-     * @return string[]
+     * @return ErrorMessage[]
```

Comment:
That makes using results a bit more complicated.

Reply:
It's almost the same thanks to `__toString()` method inside `ErrorMessage` and thanks to it `TranslatorDecorator` can translate error messages